### ISSUE TITLE
inertia:finish event.detail is null

### ISSUE THREAD

**Issue by sefaYavuz** [inertiajs/inertia#342]
I'm trying to make the `inertiajs/progress` package work. But I'm getting this error whenever I try to click on a link.

```
Uncaught TypeError: Cannot read property 'visit' of null
    at HTMLDocument.s (app.js:122)
    at Object.fireEvent (app.js:109)
    at app.js:109
    at <anonymous>
```

I've looked in the code and came across this piece of code:

```
document.addEventListener('inertia:finish', finish)

function finish(event) {
  clearTimeout(timeout)
  if (!NProgress.isStarted()) {
    return
  } else if (event.detail.visit.completed) {
    NProgress.done()
  } else if (event.detail.visit.interrupted) {
    NProgress.set(0)
  } else if (event.detail.visit.cancelled) {
    NProgress.done()
    NProgress.remove()
  }
}
```

So I've logged the output of `event` and as you can see the property `detail` is `null`.

```
bubbles: false
cancelBubble: false
cancelable: false
composed: false
currentTarget: null
defaultPrevented: false
detail: null
eventPhase: 0
isTrusted: false
path: (2) [document, Window]
returnValue: true
srcElement: document
target: document
timeStamp: 4969.599999996717
type: "inertia:finish"
__proto__: CustomEvent
```

Anyone that can help me with this problem?

**Comment by morris14** [comment]
@sefaYavuz looks like this should be cleared up by this commit:

https://github.com/inertiajs/inertia/commit/eca9e97e0e6d5e957e17a9d1e214d37e3a7621f7#diff-467a987931305e1cb6da2436149dad1d780e881e4df994731a88c53bbdf53b08R244

Not sure when this will be released though!

**Comment by reinink** [comment]
What version of Inertia.js are you on? This was released in [v0.6.0](https://github.com/inertiajs/inertia/releases/tag/v0.6.0).

**Comment by reinink** [comment]
Closing this due to lack of response. Feel free to share additional information if this issue persists, and we can reopen. I suspect this is just a versioning issue though. Make sure you're on the latest versions of Inertia, and that should clear things up.
